ksqldb-functional-tests/src/test/resources/query-validation-tests/binary-comparison.json

```diff
@@ -6,31 +6,31 @@
     {
       "name": "equals",
       "statements": [
-        "CREATE STREAM INPUT (A INT KEY, B BOOLEAN, C BIGINT, D DOUBLE, E DECIMAL(4,3), F STRING) WITH (kafka_topic='test_topic', value_format='JSON');",
-        "CREATE STREAM OUTPUT AS SELECT A, A = 1, B = true, C = 11, D = 1.1, E = 1.20, F = 'foo' FROM INPUT;"
+        "CREATE STREAM INPUT (A INT KEY, B BOOLEAN, C BIGINT, D DOUBLE, E DECIMAL(4,3), F STRING, G ARRAY<INT>, H MAP<STRING, INT>, I STRUCT<ID INT>) WITH (kafka_topic='test_topic', value_format='JSON');",
```

Comment:
thanks for updating these tests too! we'll need to regenerate the historical plans. you can do this by running `PlannedTestGeneratorTest` (and uncomment `@Ignore`)

Reply:
Ready @agavra